openslides/core/static/templates/core/customslide-list.html

```diff
@@ -48,15 +48,97 @@ <h1 translate>Custom slides</h1>
     </table>
   </div>
   <div class="col-sm-4">
-    <!-- projector live view -->
     <div class="well">
-      <h4 translate>Projector live view</h4>
+
+      <!-- projector live view -->
+      <h3 translate>Projector</h4>
       <a ui-sref="projector" target="_blank">
         <div id="iframewrapper">
           <iframe id="iframe" src="/projector" frameborder="0"></iframe>
           <div id="iframeoverlay"></div>
         </div>
       </a>
+      <!-- TODO: add functions for projector control buttons -->
+      <div os-perms="core.can_manage_projector">
+        <a os-perms="core.can_manage_projector"
+            class="btn btn-default btn-sm disabled"
+            title="{{ 'Edit current slide' | translate}}">
```

Comment:
This is not easy for we might have multiple projector elements activated. Loop over all active projector elements and add all links according to the name of the elements. Filter non editable elements like clock and countdown (but not message).